README.md

```diff
@@ -1 +1,31 @@
-# Code-review
```

Comment:
Nice job so far.
However, there are some changes that needed for your code to look professional.
[REQUIRED]The title of the project is repeated in the README.md file. Why don't you remove the first line to make the README look more professional and informative.